x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/AbstractEsqlIntegTestCase.java

```diff
@@ -53,11 +55,24 @@ public void ensureBlocksReleased() {
             CircuitBreaker reqBreaker = breakerService.getBreaker(CircuitBreaker.REQUEST);
             try {
                 assertBusy(() -> {
-                    logger.info("running tasks: {}", client().admin().cluster().prepareListTasks().get());
+                    logger.info(
+                        "running tasks: {}",
+                        client().admin()
+                            .cluster()
+                            .prepareListTasks()
+                            .get()
+                            .getTasks()
+                            .stream()
+                            .filter(
+                                t -> false == t.action().contains(TransportListTasksAction.TYPE.name())
+                                    && false == t.action().contains(HealthNode.TASK_NAME)
+                            )
+                            .toList()
+                    );
                     assertThat("Request breaker not reset to 0 on node: " + node, reqBreaker.getUsed(), equalTo(0L));
                 });
             } catch (Exception e) {
-                assertThat("Request breaker not reset to 0 on node: " + node, reqBreaker.getUsed(), equalTo(0L));
+                throw new RuntimeException("failed waiting for breakers to clear", e);
```

Comment:
I was looking at these exception a *ton* and wanted to shorten the logs that come out of the task list - to skip the ones we know we don't care about.